modules/caddytls/connpolicy.go

```diff
@@ -298,6 +311,13 @@ type ClientAuthentication struct {
 	// which are not in this list will be rejected.
 	TrustedLeafCerts []string `json:"trusted_leaf_certs,omitempty"`
 
+	//List of client certificate validators used to verify the cert
+	//Validators can be used to add additional checks like revocation checks
+	ValidatorsRaw []json.RawMessage `json:"validators,omitempty" caddy:"namespace=tls.client_cert_validators inline_key=validator"`
+
+	//Decoded Validators
+	Validators []ClientCertValidator `json:"-"`
```

Comment:
```suggestion
	// A list of client certificate validators, for additional
	// verification. These can perform custom checks, like ensuring
	// the certificate is not revoked.
	ValidatorsRaw []json.RawMessage `json:"validators,omitempty" caddy:"namespace=tls.client_cert_validators inline_key=validator"`

	Validators []ClientCertValidator `json:"-"`
```

Comments should have a space at the start. They get rendered as markdown in our docs on the website, so they should have punctuation. Adjusted the wording a bit.

Also, I'm not sure if we need `Validators` to be exported. If it's not used outside of this package, we can change it to `validators`. The empty `json` field tag is not needed in that case.